packages/roosterjs-editor-plugins/lib/plugins/Paste/wordConverter/wordOnlineTableConverter.ts

```diff
@@ -0,0 +1,24 @@
+import { IEditor } from 'roosterjs-editor-types';
+import { VTable } from 'roosterjs-editor-dom';
+
+/**
+ * @internal
+ * Fix styles of table copied from word online
+ * @param editor
+ * @param table copied from word online
+ */
+export default function wordOnlineTableConverter(editor: IEditor, table: HTMLTableElement) {
+    const vTable = new VTable(table as HTMLTableElement);
```

Comment:
Quick question, is it possible to just use querySelectorAll('td,th') or querySelectorAll('p') in this function and edit the style if needed. That way we don't need to build the VTable object.